lib/action_logic/action_validation.rb

```diff
@@ -6,27 +6,31 @@ module ActionLogic
   module ActionValidation
     module ClassMethods
       def validates_before(args)
-        @validates_before = args
+        @validates_before = args.merge(raise_action_logic_exception: true)
       end
 
       def validates_after(args)
-        @validates_after = args
+        @validates_after = args.merge(raise_action_logic_exception: true)
```

Comment:
I don't know how I feel about using an extra attribute to signal the validation classes. This is just a rough draft. If this is the approach to take, it needs to be a very specific name to avoid potential collisions, hence "raise_action_logic_exception".